### DISCUSSION TITLE
Title FontSize

### DISCUSSION THREAD

**Discussion by jomimota** [ofekashery/vertical-stack-in-card#149] [Q&A]
Hi,

It's possible to change / reduce the fontsize of title text?
I can center this text text via card-mod, like this:

        card_mod:
          style: |
            ha-card {
              text-align: center;
            }

but I want to change the fontsize.

Thanks in Advance.

**Comment by droidgren** [comment]
In case someone still wondering...

```
    card_mod:
      style: |
        ha-card {
         text-align: center;
          --ha-card-header-font-size: 1.5em;

        }
```
